forage-android/src/main/java/com/joinforage/forage/android/network/model/ForageApiResponse.kt

```diff
@@ -6,9 +6,23 @@ internal val UnknownErrorApiResponse = ForageApiResponse.Failure.fromError(
     ForageError(500, "unknown_server_error", "Unknown Server Error")
 )
 
+/**
+ * A model that represents the possible types of responses from the Forage API.
+ *
```

Comment:
Can remove extra `*` newline on line 11

Reply:
Yes of course! I had hoped the spotlessApply would've caught these things 🤔